tools/xcodeproj_shims/install.sh

```diff
@@ -9,40 +9,50 @@ set -eux
 find "${BAZEL_DIAGNOSTICS_DIR}" -type f -atime +7d -delete
 
 case "${PRODUCT_TYPE}" in
-    com.apple.product-type.framework)
-        input_options=("bazel-bin/$BAZEL_BIN_SUBDIR/${TARGET_NAME}/${FULL_PRODUCT_NAME}")
-        ;;
-    com.apple.product-type.framework.static)
-	    # For static library, as the output is not under bazel-bin, we have to get it based on build event
-        # Example of such entry inside build event:
-        # important_output {
-        #   name: ".../SomeFramework.framework/SomeFramework"
-        #   uri: "file:///private/var/tmp/.../.../SomeFramework.framework/SomeFramework"
-        #   path_prefix:... 
-        #   ...
-        #  } 
-        # We only care about the entry ending with TARGET_NAME" so that we can get the path to its directory
-        QUERY=`grep -A 2 important_output "$BAZEL_BUILD_EVENT_TEXT_FILENAME" | grep -w uri | grep ${TARGET_NAME}\" | sed "s/uri: \"file:\/\///"`
-        if [[ -z $QUERY ]]; then 
-            echo "Unable to locate resource for framework of ${TARGET_NAME}"
-            exit 1
-        fi
-        input_options=($(dirname "${QUERY}"))
-        ;;
-    com.apple.product-type.bundle.unit-test)
-        input_options=("bazel-bin/$BAZEL_BIN_SUBDIR/${FULL_PRODUCT_NAME}" "bazel-bin/$BAZEL_BIN_SUBDIR/$TARGET_NAME.__internal__.__test_bundle_archive-root/$TARGET_NAME${WRAPPER_SUFFIX:-}")
-        ;;
-    com.apple.product-type.application)
-        input_options=("bazel-bin/$BAZEL_BIN_SUBDIR/${FULL_PRODUCT_NAME}" "bazel-bin/$BAZEL_BIN_SUBDIR/${TARGET_NAME}_archive-root/Payload/$TARGET_NAME${WRAPPER_SUFFIX:-}")
-        ;;
-    *)
-        echo "Error: Installing ${TARGET_NAME} of type ${PRODUCT_TYPE} is unsupported" >&2
+com.apple.product-type.framework)
```

Comment:
nit: add indent?

Reply:
this is the indentation that `shfmt` wants